packages/storybook/src/nimble/anchor-button/anchor-button-matrix.stories.ts

```diff
@@ -57,7 +57,7 @@ const component = (
     </${anchorButtonTag}>
 `;
```

Comment:
Could you apply the guidance to Spright tests too? For example I noticed that Rectangle is still including its component name in the Theme Matrix story.